recipe/meta.yaml

```diff
@@ -21,11 +21,8 @@ requirements:
     - {{ compiler('fortran') }}
     - {{ compiler('c') }}
   host:
-    - libnetcdf
-    - hdf5
-  run:
-    - libnetcdf
-    - hdf5
+    - libnetcdf <=4.6.1
+    - hdf5 <=1.10.3
```

Comment:
Why `libnetcdf<=4.6.1` and `hdf5 <=1.10.3`? If this pin is unavoidable we should be conservative and use strict pins to `4.6.1` and `1.10.3` dropping the `<=`.

Reply:
This is a patch around the upstream bug that I plan to remove. I put the pinnings in so that I could test the build. Once the upstream package releases a bug fix, we can remove this. We should not merge until that happens or we apply a patch locally.